doc/OnlineDocs/README.txt

```diff
@@ -22,6 +22,10 @@ invoke 'make' differently.  For example, using the PyUtilib 'lbin' command:
 
   lbin make html
 
+NOTE: If you get the error ``ModuleNotFoundError: No module named 'sphinx_rtd_theme'``, try running:
+
+  pip install sphinx_rtd_theme
+
```

Comment:
Can you just add this to step 0 so that it is:
```
pip install sphinx sphinx_rtd_theme
```

Reply:
Good suggestion. I just pushed the change.